ui-v2/src/api/task-run-concurrency-limits/task-run-concurrency-limits.ts

```diff
@@ -72,6 +72,10 @@ export const buildDetailTaskRunConcurrencyLimitsQuery = (id: string) =>
 		queryFn: () => fetchTaskRunConcurrencyLimit(id),
 	});
 
+// Alias for consistency with other query factory naming conventions
+export const buildGetConcurrencyLimitQuery =
+	buildDetailTaskRunConcurrencyLimitsQuery;
+
```

Comment:
You don't need to alias this. You can just rename it and update all the uses in the app.